src/net/sourceforge/kolmafia/request/PlaceRequest.java

```diff
@@ -339,6 +348,29 @@ else if (responseText.contains("a giant jagged hole in the wall")) {
     SpadingManager.processPlace(urlString, responseText);
   }
 
+  private static void parseSotVisit(String responseText) {
+    String location = "";
```

Comment:
I mean, whatever. I'd just use null and != rather than !xxx.equals(). Why force a method call?

Reply:
You're right.  Using a string was leftover from when I thought I needed to compare the location just found to the current value of the preference.  I don't do that so yes, null is a better check.